src/Protocol/Authenticator.cpp

```diff
@@ -223,7 +223,7 @@ bool cAuthenticator::GetPlayerProperties(const AString & a_UUID, Json::Value & a
 	AString Request;
 	Request += "GET " + ActualAddress + " HTTP/1.0\r\n";
 	Request += "Host: " + m_Server + "\r\n";
-	Request += "User-Agent: MCServer\r\n";
+	Request += "User-Agent: Cuberite\r\n";
```

Comment:
@worktycho Said that we should use something like

    Cuberite (MCServer)

here. I'm not quite sure of the reason behind it though.